src/planet-a/SubdivTriangleMesh.cpp

```diff
@@ -0,0 +1,170 @@
+/**
+ * Open Space Program
+ * Copyright © 2019-2021 Open Space Program Project
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include "SubdivTriangleMesh.h"
+
+#include <Magnum/Math/Vector3.h>
+
+using namespace planeta;
+
+
+
+ChunkedTriangleMeshInfo planeta::make_subdivtrimesh_general(
+        unsigned int chunkMax, unsigned int subdivLevels, int pow2scale)
+{
+    // calculate stuff here
+
+    using std::sqrt;
```

Comment:
Why are you pulling these into the namespace? This applies to the entire namespace, not just this function.

Better to just use std:: when you call the functions.

Reply:
> Why are you pulling these into the namespace?
Unless something really weird is happening, I don't think this is this is the case. I added a `using std::make_unique;` there as an example, and tried accessing from next the function below, where it wasn't recognized.

Extra noise like "std::" can make reading equations really tedious. It's a good reason why mathematicians like single letters and symbols.